components/Lessons/Lessons.module.css

```
@@ -1,3 +1,10 @@
-.main {
+.container {
   background-color: white;
+  display: grid;
+  grid-template-rows: auto auto 1fr auto;
+  align-content: start;
+  height: 100vh;
```

```suggestion
  height: 100%;
```
nesting vhs is not always a good idea because this can create unwanted overflows

---

Does not seems to work here with „%".